grafana-plugin/src/components/GForm/GForm.types.ts

```diff
@@ -13,6 +14,7 @@ export interface FormItem {
   type: FormItemType;
   description?: string;
   normalize?: (value: any) => any;
+  shouldShow?: (data: any) => any;
```

Comment:
minor: `hidden` or `visible` naming looks more _standartish_

Reply:
I'll change to `isVisible`